Makefile

```diff
@@ -0,0 +1,28 @@
+GO ?= go
+GOLANGCI_LINT ?= golangci-lint
+GOLANGCI_LINT_VERSION ?= v1.29.0
+
+.PHONY: all
+all: lint vet test-race
+
+.PHONY: lint
+lint: linter
+	$(GOLANGCI_LINT) run
+
+.PHONY: linter
+linter:
+	which $(GOLANGCI_LINT) || ( cd /tmp && GO111MODULE=on $(GO) get -u github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) )
```

Comment:
It appeared that binary install with upstream install.sh works much more reliably https://github.com/ethersphere/bee/blob/master/Makefile#L36.